skimage/feature/corner.py

```diff
@@ -455,7 +455,7 @@ def shape_index(image, sigma=1, mode='constant', cval=0):
 
     Parameters
     ----------
-    image : ndarray
+    image : 2D array
```

Comment:
```suggestion
    image : (M, N) ndarray
```
I think this should probably be `(M, N) ndarray`, but haven't tracked down the issue where we discussed the style to use previously.

None of the functions in this `corner.py` file use that style, but many others in the library do (See for example `haar.py`, `match.py`, `orb.py` in this same folder)

Reply:
Thanks for the suggestion!

Reply:
I can not either remember if we already specified the style to use, but I think that you are right @grlee77: the style you are suggesting is already used in many places in the repo.

Reply:
Right. Apparently, we also have:
https://github.com/scikit-image/scikit-image/blob/3ad36415ae24ad5322e0ecf2d507bac781ded5dc/skimage/morphology/gray.py#L25
in some places. Let's stick with `(M, N) ndarray`.